docs/content/docs/tutorials/memorydb-example.md

```diff
@@ -0,0 +1,288 @@
+apiVersion: memorydb.services.k8s.aws/v1alpha1
+kind: SubnetGroup
+metadata:
+  name: "${MEMORYDB_SUBNETGROUP_NAME}"
+spec:
+  name: "${MEMORYDB_SUBNETGROUP_NAME}"
+  description: "MemoryDB cluster subnet group"
+  subnetIDs:
+$(printf "    - %s\n" ${SUBNET_IDS})
+
+EOF
+
+kubectl apply -f memorydb-subnetgroup.yaml
+kubectl describe subnetgroup "${MEMORYDB_SUBNETGROUP_NAME}"
+```
+##### Create MemoryDB Cluster
+The following example uses the MemoryDB Subnet Group created above. It uses EKS Cluster Security group. You may use any other VPC security group as well.
+
+```shell
+EKS_CLUSTER_NAME="<your EKS cluster name>"
+MEMORYDB_CLUSTER_NAME="<your MemoryDB cluster name>"
+SECURITY_GROUP_ID=$(aws eks describe-cluster --name $EKS_CLUSTER_NAME --query cluster.resourcesVpcConfig.clusterSecurityGroupId)
+
+cat <<EOF > memorydb-cluster.yaml
+apiVersion: memorydb.services.k8s.aws/v1alpha1
+kind: Cluster
+metadata:
+  name: "${MEMORYDB_CLUSTER_NAME}"
+spec:
+  name: "${MEMORYDB_CLUSTER_NAME}"
+  nodeType: db.t4g.small
+  aclName: open-access
+  securityGroupIDs:
+    - SECURITY_GROUP_ID
+  subnetGroupName: ${MEMORYDB_SUBNETGROUP_NAME}
+EOF
+
+kubectl apply -f memorydb-cluster.yaml
+kubectl describe cluster "${MEMORYDB_CLUSTER_NAME}"
+```
+
+You can track the status of the provisioned database using `kubectl describe` on the `Cluster` custom resource:
+
+```bash
+kubectl describe cluster "${MEMORYDB_CLUSTER_NAME}"
+```
+
+When the `Cluster Status` says `Available`, you can connect to the database instance.
+
+## Connect to MemoryDB Cluster
+
+>To connect to the MemoryDB Cluster from a Pod running inside Kubernetes cluster:
+> * Ensure that the [Amazon VPC Container Network Interface (CNI) plugin for Kubernetes](https://docs.aws.amazon.com/eks/latest/userguide/managing-vpc-cni.html) has been setup for the EKS Cluster.
+> * Review [access patterns for accessing a MemoryDB Cluster in an Amazon VPC](https://docs.aws.amazon.com/memorydb/latest/devguide/memorydb-vpc-accessing.html) to confirm that MemoryDB cluster is configured to allow connection from the Pod.
+
+The `Cluster` status contains the information for connecting to a MemoryDB Cluster. The host information can be found in `status.ClusterEndpoint.Address` and the port information can be found in `status.ClusterEndpoint.Port`. For example, you can get the connection information for a `Cluster` created in one of the previous examples using the following commands:
+
+```bash
+kubectl get cluster "${MEMORYDB_CLUSTER_NAME}" -o jsonpath='{.status.ClusterEndpoint.Address}'
+kubectl get cluster "${MEMORYDB_CLUSTER_NAME}" -o jsonpath='{.status.ClusterEndpoint.Port}'
+```
+
+
+
+You can extract this information and make it available to your Pods using a [`FieldExport`][field-export] resource. For example, to get the connection information from MemoryDB Cluster created the above example, you can use the following example:
+
+```shell
+MEMORYDB_CLUSTER_NAME="<your cluster name>"
```

Comment:
Like `EKS_CLUSTER_NAME`, lets re-use this value from above

Reply:
I preferred keeping the relevant variables close to the commands.